mmv1/third_party/terraform/resources/resource_dataproc_cluster.go.erb

```diff
@@ -638,6 +656,34 @@ func resourceDataprocCluster() *schema.Resource {
 											},
 										},
 									},
+
+									"reservation": {
```

Comment:
Thanks Ryan pointing out this. I initially get confused to follow the gcloud cli or the API, since i saw the subnet_uri and network_uri followed the gcloud format, so i chose follow the gcloud cli as well. 

Since derive from API is prefered, i updated the schema based on the API.